float64 altitude
---
bool success
